docs/arch.md

```diff
@@ -138,6 +144,26 @@ health_statuses_insights{metric="moderate"} 1
 health_statuses_insights{metric="total"} 2
 ```
 
+### Metrics
+
+- `health_statuses_insights`, information about the cluster health status as detected by Insights tooling
```

Comment:
It would be good to extend the description here a little bit - what information? What the value of the metric represents.